SegmentEditorFastMarching/SegmentEditorFastMarchingLib/SegmentEditorEffect.py

```diff
@@ -125,31 +125,31 @@ def percentMaxChanged(self, val):
   def fastMarching(self,percentMax):
 
     self.fm = None
-    
-    # Get master volume image data
+
+    # Get source volume image data
     import vtkSegmentationCorePython as vtkSegmentationCore
-    masterImageData = self.scriptedEffect.masterVolumeImageData()
+    sourceImageData = self.scriptedEffect.sourceVolumeImageData()

```

Comment:
Keep backward compatibility with Slicer-5.0 to make it easier to keep the extension up-to-date for both stable and preview releases.